apps/web/tests/activity-graph.spec.ts

```diff
@@ -20,6 +26,6 @@ test('should be able to add a new channel', async ({ page }) => {
   await page.goto('/activities');
   await expect(page).toHaveURL(/\/activities/);
 
-  const addChannelButton = getByTestId(page, 'activity-stats-weekly-sent');
+  const addChannelButton = page.getByTestId('activity-stats-weekly-sent');
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do not leave false praise (which can actually be damaging). Do look for something to sincerely praise.  -->
👏 **praise:** love this approach!